src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/ObjectWriter/Dwarf/DwarfEhFrame.cs

```diff
@@ -55,33 +55,33 @@ private void WriteCie(DwarfCie cie)
 
             if (cie.FdesHaveAugmentationData)
             {
-                augmentationString.Append('z');
+                augmentationString.Append((byte)'z');
             }
             if (cie.PersonalitySymbolName != null)
             {
-                augmentationString.Append('P');
+                augmentationString.Append((byte)'P');
                 augmentationLength += 1u + AddressSize(cie.PersonalityEncoding);
             }
             if (cie.LsdaEncoding != 0)
             {
-                augmentationString.Append('L');
+                augmentationString.Append((byte)'L');
                 augmentationLength++;
             }
             if (cie.PointerEncoding != 0)
             {
-                augmentationString.Append('R');
+                augmentationString.Append((byte)'R');
                 augmentationLength++;
             }
             if (cie.IsSignalFrame)
             {
-                augmentationString.Append('S');
+                augmentationString.Append((byte)'S');
```

Comment:
nit: for style consistency, we could use ""u8 string for single chars as well (since `"S"u8.Length` => 1)

Reply:
It's gonna be a lot of IL for but will do 👍

Reply:
It's gonna be slower since it would involve a .CopyTo should the method not be inlined. Please prefer using chars/bytes over using single-element literals.

Reply:
Does JIT fold ``[cns].CopyTo(dst)``? We could slap AI on the ROS<byte> append if it does 🤔

Reply:
It does but CopyTo/Memmove unrolling is still more heavy than just `ptr = value; _length++`. I just don't think there's a need to use `"c"u8` here, you can also add a guard to .Append(char c) and cast it there, It doesn't usually matter but in e.g. a loop the cost accumulates.

Reply:
Neither this is a performance critical path nor it produces a huge diff (which can be optimized by JIT for length:1 someday).